deploy/bundle.yaml

```diff
@@ -308,30 +308,49 @@ spec:
   replicas: 1
   selector:
     matchLabels:
-      name: percona-xtradb-cluster-operator
+      app.kubernetes.io/component: operator
+      app.kubernetes.io/instance: percona-xtradb-cluster-operator
+      app.kubernetes.io/name: percona-xtradb-cluster-operator
+      app.kubernetes.io/part-of: percona-xtradb-cluster-operator
+  strategy:
+    rollingUpdate:
+      maxUnavailable: 1
+    type: RollingUpdate
   template:
     metadata:
       labels:
-        name: percona-xtradb-cluster-operator
+        app.kubernetes.io/component: operator
+        app.kubernetes.io/instance: percona-xtradb-cluster-operator
+        app.kubernetes.io/name: percona-xtradb-cluster-operator
+        app.kubernetes.io/part-of: percona-xtradb-cluster-operator
     spec:
-      serviceAccountName: percona-xtradb-cluster-operator
       containers:
-        - name: percona-xtradb-cluster-operator
-          image: percona/percona-xtradb-cluster-operator:1.6.0
-          ports:
-          - containerPort: 60000
-            name: metrics
-          command:
-          - percona-xtradb-cluster-operator
-          imagePullPolicy: Always
-          env:
-            - name: WATCH_NAMESPACE
-              valueFrom:
-                fieldRef:
-                  fieldPath: metadata.namespace
-            - name: POD_NAME
-              valueFrom:
-                fieldRef:
-                  fieldPath: metadata.name
-            - name: OPERATOR_NAME
-              value: "percona-xtradb-cluster-operator"
+      - command:
+        - percona-xtradb-cluster-operator
+        env:
+        - name: WATCH_NAMESPACE
+          valueFrom:
+            fieldRef:
+              apiVersion: v1
+              fieldPath: metadata.namespace
+        - name: POD_NAME
+          valueFrom:
+            fieldRef:
+              apiVersion: v1
+              fieldPath: metadata.name
+        - name: OPERATOR_NAME
+          value: percona-xtradb-cluster-operator
+        image: percona/percona-xtradb-cluster-operator:1.6.0
+        imagePullPolicy: Always
+        livenessProbe:
+          failureThreshold: 3
+          httpGet:
+            path: /metrics
+            port: metrics
+            scheme: HTTP
```

Comment:
This breaks all deployments because the operator doesn't actually respond on either port 8080 or port 60000. Was this functionality ever implemented in the operator image? How did it get into this manifest if it doesn't work?